<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Navigation</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="first_pushButton">
         <property name="text">
          <string>first</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="previous_pushButton">
         <property name="text">
          <string>previous</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="last_pushButton">
         <property name="text">
          <string>last</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="next_pushButton">
         <property name="text">
          <string>next</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="5">
     <widget class="QTableView" name="tableView">
      <property name="minimumSize">
       <size>
        <width>650</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="string_view_stackedWidget">
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="contract_table_page">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="contract_record_groupBox">
          <property name="title">
           <string>Current Record:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="1" colspan="3">
            <widget class="QSpinBox" name="device_amount_spinBox">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QSpinBox" name="contracts_number_spinBox">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="maximum">
              <number>100000</number>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>start day</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>device </string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="3">
            <widget class="QComboBox" name="customer_head_name_comboBox"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>customer</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>amount</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>number</string>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="QPushButton" name="delete_contract_pushButton">
             <property name="text">
              <string>DELETE</string>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QPushButton" name="create_contract_PushButton">
             <property name="text">
              <string>NEW</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="3">
            <widget class="QDateEdit" name="date_of_the_agreement_dateEdit"/>
           </item>
           <item row="6" column="1" colspan="3">
            <widget class="QDateEdit" name="deadline_dateEdit"/>
           </item>
           <item row="1" column="1" colspan="3">
            <widget class="QComboBox" name="device_name_comboBox"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>deadline day</string>
             </property>
            </widget>
           </item>
           <item row="8" column="2">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="2">
         <widget class="QPushButton" name="delete_customer_pushButton">
          <property name="text">
           <string>DELETE</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Current Record:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="1">
            <widget class="QLineEdit" name="customer_address_lineEdit">
             <property name="maxLength">
              <number>30</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>checking account</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="customer_checking_account_lineEdit">
             <property name="maxLength">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>telephone</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>head_name</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="customer_telephone_ineEdit">
             <property name="maxLength">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="customer_head_name_lineEdit">
             <property name="maxLength">
              <number>30</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>address</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="create_customer_pushButton">
          <property name="text">
           <string>NEW</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="3" column="2">
         <widget class="QPushButton" name="delete_device_pushButton">
          <property name="text">
           <string>DELETE</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="create_device_pushButton">
          <property name="text">
           <string>NEW</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Current Record:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="3" column="1">
            <widget class="QSpinBox" name="device_price_spinBox">
             <property name="maximum">
              <number>100000000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>name</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>price</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="device_name_lineEdit">
             <property name="maxLength">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>code</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="device_code_lineEdit">
             <property name="maxLength">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="1" column="2">
         <widget class="QPushButton" name="delete_team_member_pushButton">
          <property name="text">
           <string>DELETE</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="create_team_member_pushButton">
          <property name="text">
           <string>NEW</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Current record:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>employee</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>position</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>project</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QComboBox" name="team_position_comboBox"/>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QComboBox" name="team_employee_comboBox"/>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QComboBox" name="team_project_number_comboBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="1" column="2">
         <widget class="QPushButton" name="delete_employee_pushButton">
          <property name="text">
           <string>DELETE</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Current record:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="1" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>name</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="employee_name_lineEdit">
             <property name="maxLength">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="employee_code_lineEdit">
             <property name="maxLength">
              <number>6</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>code</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="create_employee_pushButton">
          <property name="text">
           <string>NEW</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="0" column="0" colspan="3">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Current record:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>code</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="position_code_lineEdit">
             <property name="maxLength">
              <number>6</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>name</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="position_name_lineEdit">
             <property name="maxLength">
              <number>20</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="create_position_pushButton">
          <property name="text">
           <string>NEW</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="delete_position_pushButton">
          <property name="text">
           <string>DELETE</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1047</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen_Table">
    <property name="title">
     <string>Open Table</string>
    </property>
    <addaction name="actionContracts"/>
    <addaction name="actionCustomers"/>
    <addaction name="actionDevices"/>
    <addaction name="actionTeam_members"/>
    <addaction name="actionEmployees"/>
    <addaction name="actionPositions"/>
   </widget>
   <addaction name="menuOpen_Table"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionContracts">
   <property name="text">
    <string>contracts</string>
   </property>
  </action>
  <action name="actionCustomers">
   <property name="text">
    <string>customers</string>
   </property>
  </action>
  <action name="actionDevices">
   <property name="text">
    <string>devices</string>
   </property>
  </action>
  <action name="actionEmployees">
   <property name="text">
    <string>employees</string>
   </property>
  </action>
  <action name="actionPositions">
   <property name="text">
    <string>positions</string>
   </property>
  </action>
  <action name="actionTeam_members">
   <property name="text">
    <string>team members</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
